recovery-init

```diff
@@ -34,39 +104,44 @@ mkdir /dev/pts
 mount -a
 cat /proc/mounts > /etc/mtab
 
+mkdir /config
+mount -t configfs none /config
+
 #Run mdev
 echo "Running mdev..."
 mdev -s
+
+
 #Set mdev as hotplug manager
 echo /bin/mdev > /proc/sys/kernel/hotplug
 
-write()
-{
-	echo -n "$2" > $1
-}
 
 # Minimize power consumption by lowering display brightness to minimum
 write $DISPLAY_BRIGHTNESS_PATH $DISPLAY_BRIGHTNESS
 
-write /sys/class/android_usb/android0/enable 0
-write /sys/class/android_usb/android0/idVendor 2931
-write /sys/class/android_usb/android0/idProduct 0A06
-write /sys/class/android_usb/android0/functions rndis,mass_storage
-write /sys/class/android_usb/android0/f_rndis/ethaddr 00:01:02:03:04:05
-write /sys/class/android_usb/android0/enable 1
-write /sys/class/android_usb/android0/iManufacturer Jolla
-write /sys/class/android_usb/android0/iProduct Recovery
-write /sys/class/android_usb/android0/iSerial 123456789
-write /sys/class/android_usb/android0/enable 1
+
+
+usb_setup "Jolla Debug setting up"
 
 ip link set rndis0 up
-ipaddr add 10.42.66.66/29 broadcast 10.42.66.255 dev rndis0
-ipaddr add 192.168.2.15/24 broadcast 192.168.2.255 dev rndis0 label rndis0:0
+if ipaddr add 10.42.66.66/29 broadcast 10.42.66.255 dev rndis0 ; then
```

Comment:
Noticed still one functional difference compared to old release when testing. Could you add following line after then and before USB_IFACE=rndis0:
ipaddr add 192.168.2.15/24 broadcast 192.168.2.255 dev rndis0 label rndis0:0

Reply:
I have already removed that line. Please pull a gain.

Reply:
Fixed that but used $USB_IFACE instead of static rndis0.

Reply:
`ip link set rndis0 up
if ipaddr add 10.42.66.66/29 broadcast 10.42.66.255 dev rndis0 ; then
    ipaddr add 192.168.2.15/24 broadcast 192.168.2.255 dev rndis0 label rndis0:0
    USB_IFACE=rndis0
elif ipaddr add 10.42.66.66/29 broadcast 10.42.66.255 dev usb0 ; then`

Reply:
Sorry, meant like that ^